src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/AssignTaskTest.java

```diff
@@ -49,6 +49,22 @@
     private AccessControlResponse assigneeAccessControlResponse;
     private String taskId;
 
+    private static Stream<Scenario> provideScenario() {
```

Comment:
👍 this sounds familiar

Reply:
This is your code I think the formatting moved it up :)